eng/helix/content/runtests.cmd

```diff
@@ -17,15 +17,18 @@ shift
 set $helixTimeout=%9
 
 set DOTNET_HOME=%HELIX_CORRELATION_PAYLOAD%\sdk
-set DOTNET_ROOT=%DOTNET_HOME%\%$arch%
+set DOTNET_ROOT=%HELIX_CORRELATION_PAYLOAD%\dotnet
```

Comment:
This is probably breaking IIS tests. 

The error coming from IIS is:
```
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] Could not find dotnet.exe at 'C:\h\w\AED609D4\p\sdk\dotnet.exe'' 
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] End Event Log Message.
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] InvalidOperationException 'Could not find dotnet.exe at 'C:\h\w\AED609D4\p\sdk\dotnet.exe'' caught at F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\HostFxrResolutionResult.cpp:64 
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] Failed HRESULT returned: 0x8007023e at F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\HandlerResolver.cpp:68 
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] Failed HRESULT returned: 0x8007023e at F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\HandlerResolver.cpp:171 
[301.828s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] Failed HRESULT returned: 0x8007023e at F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\applicationinfo.cpp:181 
[301.829s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] Failed HRESULT returned: 0x8007023e at F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\applicationinfo.cpp:96 
[301.829s] [Microsoft.AspNetCore.Server.IntegrationTesting.IIS.IISDeployer] [Information] [aspnetcorev2.dll] Event Log: 'Failed to start application '/LM/W3SVC/1/ROOT', ErrorCode '0x8007023e'.' 
```

I think it may be using DOTNET_HOME to find dotnet, I think you'll need to update https://github.com/dotnet/aspnetcore/blob/9a1810c1dbe432fc7bc7e8bc68fa22ab787c0452/src/Hosting/Server.IntegrationTesting/src/Common/DotNetCommands.cs.

Reply:
Ah cool that's why I didn't find it, i was searching under servers not hosting... thanks for pointing this out

Reply:
I'm going to try removing dotnet_home on helix to see if that works first

Reply:
actually I'll just set it to the DOTNET_ROOT since that might be less impactful

Reply:
We don't actually set DOTNET_HOME on the non windows versions either...(runtests.sh)

Reply:
@jkotalik @Tratcher progress, looks like the tests are passing now, but it does seem like there's a sporadic hang (with a dump attached) not sure if this is new or expected from time to time?